src/main/java/org/fundacionjala/at15/katas/langtonant/adrian/Board.java

```diff
@@ -0,0 +1,48 @@
+package org.fundacionjala.at15.katas.langtonant.adrian;
+
+public class Board {
+    private int width;
+    private int height;
+    private int[][] tablero;
```

Comment:
Instead of 'tablero', you should use 'grid' or something similar. Remember keep using english for everything